docs/using/intro.md

```diff
@@ -2,10 +2,73 @@
 
 OpenFF Interchange is a Python package developed by the Open Force Field
 Initiative for storing, manipulating, and converting molecular mechanics data.
-Most importantly, the package provides the [`Interchange`] class, which stores
+The package is oriented around the [`Interchange`] class, which stores
 a molecular mechanics system and provides methods to write the system out in
 numerous formats.
 
+An `Interchange` contains a fully parameterized molecular system with all the
+information needed to start a simulation. This includes the force field, box
+vectors, positions, velocities, and a topology containing individual molecules
+and their connectivity. For most users, Interchange forms the bridge between
+the OpenFF ecosystem and their simulation software of choice; users describe
+their system with the [OpenFF Toolkit] and then parameterize it with Interchange.
+
+An `Interchange` stores parameters in handlers that link the topology to the
+force field. This allows changes in the force field to be reflected in the
+`Interchange` immediately. This is useful for iteratively tweaking parts of a
+force field without having to recompute expensive parts like the charges.
+
+Once the Interchange is created and parameterized, it can be exported as
+simulation-ready input files to a number of molecular mechanics software packages,
+including Amber, OpenMM, GROMACS, and LAMMPS.
+
+:::{mermaid}
+---
+alt: "Flowchart describing the construction and use of an Interchange (See textual description below)"
+align: center
+---
```

Comment:
Non-blocking: This is really, really cool, but I see a bit of it renders funny for me.

![image](https://user-images.githubusercontent.com/7935382/161122364-e42c35a8-0a67-4cfe-adb4-03656c53d6b9.png)

Reply:
~~This renders correctly (in the RTD build) on my desktop but not my phone. I suspect it's not finding the fonts from the rest of the page (I have them installed locally on my machine and I had to tweak the text size after changing them), but they look right. Could you tell me what font your machine is using for the "openff.interch" text, as well as the "Positions" text? Size, weight, spacing and face would be great. On firefox you can select the text, right click -> inspect, and then click the "Fonts" tab on the right (it might be hidden under the down arrow).~~

Ok I'm getting this too but only on the first load of the intro page (or when refreshing with cleared cache). Subsequent loads seem to be fine. Is that the case for you too? (so weird but you do you JavaScript)

Looks similar to a reported bug: https://github.com/mermaid-js/mermaid/issues/2651